docs/mvp-features.md

```diff
@@ -0,0 +1,50 @@
+# Voyager MVP Features
+These features of Voyager are considered essential for MVP. They are required to be completed before we can launch the Cosmos Hub.
+
+## Managing accounts
+
+The user should be able to:
+
+* Create an account with a new seed and address
+* Import an account with a Cosmos seed
+* Import an account with the seed contained in hardware (i.e. Ledger)
```

Comment:
I think this should be: Register a private key stored on hardware (i.e. Ledger)

As I think the key is not imported into the keystore.